src/middleware/github-server-app-middleware.test.ts

```diff
@@ -25,23 +29,23 @@ describe("github-server-app-middleware", () => {
 		};
 	});
 
-	it("should call next() when no gitHupAppId is provided",  async() => {
+	it("should call next() when no uuid is provided",  async() => {
 		req = {
 			log: getLogger("request"),
 			params: {
-				id: undefined
+				uuid: undefined
 			}
 		};
```

Comment:
this could be made into a function to reduce duplication

Reply:
It is a trade-off, either make refactoring on the code to make it better, or keep the code as it is to make pull request more readable. I chose the later in this case.

But if you strongly suggest I should refactor the code in this case, i can do that as well.